src/client/components/Elevator/Elevator.component.js

```diff
@@ -0,0 +1,120 @@
+  },
+  {
+    header: 'PRESENTING YOURSELF',
+    content:
+      '<h3>Craft your pitch</h3><p><span></span>Hello, my name is ____________ and I am completing a ____________ degree in ____________ at XYZ school.</p><p>I am interested in a career in (or position as a) ____________ in the ____________ field (industry).</p><p>I have involved (during college or previous job) in ____________.</p><p>And developed skills in ____________. I have also had an internship position (employment) as a ____________ with ____________ and discovered that I really enjoy ____________.</p><p>Could you tell me more about ____________?</p>',
+  },
+  {
+    header: 'PRESENTING YOURSELF',
+    content:
+      '<h5>Avoid:</h5><p>Speaking too fast.</p><p>Rambling (It&lsquo;s so important to practice your elevator speech.)</p><p>Don&lsquo;t frown, or speak in a monotone way. Here&lsquo;s one of the downsides to rehearsing: it can leave you more focused on remembering the exact words you want to use, and less on how you&lsquo;re carrying yourself. Keep your energy level high, confident, and enthusiastic.</p>',
+  },
+  {
+    header: 'EXAMPLES',
+    content:
+      '<p>I recently graduated from college with a degree in communications. I worked on the college newspaper as a reporter, and eventually, as the editor of the arts section. I&lsquo;m looking for a job that will put my skills as a journalist to work.</p><p>I create illustrations for websites and brands. My passion is coming up with creative ways to express a message, and drawing illustrations that people share on social media.</p>',
+  },
+  {
+    header: 'KEY TAKEAWAYS',
+    content:
+      '<h4>Keep it short and sweet:</h4><p>Your elevator speech is a sales pitch. Be sure you can deliver your message in 60 seconds or less.</p><h4>Focus on the essentials:</h4><p>Say who you are, what you do, and what you want to achieve.</p><h4>Be positive:</h4><p>Your time is limited. Focus on what you want to do, not what you do&lsquo;t want to do. Be upbeat and flexible.</p><h4>Practice, practice, practice:</h4><p>Deliver your speech to a friend or record it, so that you can be sure that your message is clear.</p>',
+  },
+];
+
+const ElevatorLevel = ({
+  header,
+  content,
+  floor,
+  onClickPrev,
+  onClickNext,
+}) => {
+  return (
+    <div className="elevator-level">
+      <h3 className="elevator-heading">{header}</h3>
+      <div
+        className="elevator-content"
+        dangerouslySetInnerHTML={{ __html: content }}
+      />
+      {floor > 0 && floor <= 5 ? (
+        <img
+          src={UpArrow}
+          alt="go to previous"
+          onClick={onClickPrev}
+          aria-hidden="true"
+          className="elevator-arrow elevator-up"
```

Comment:
Hi @TITHI1244 , I think we are supposed to show up arrow when we are at floor 0, but you are showing down arrow, and the onclick functions you mistakenly used onclickprev for up arrow, onclicknext for down arrow, and also alt for arrows(in reverse), i think it is supposed to be 
`{floor > 0 && floor <= 5 ? (
        <img
          src={DownArrow}
          alt="go to next"
          onClick={onClickPrev}
          aria-hidden="true"
          className="elevator-arrow elevator-down"
        />
      ) : (
          <div />
        )}
      {floor >= 0 && floor < 5 ? (
        <img
          src={UpArrow}
          alt="go to previous"
          onClick={onClickNext}
          aria-hidden="true"
          className="elevator-arrow elevator-up"
        />`, 
Please let me know if i am wrong

Reply:
Hi Rajani. Thanks for your comment. But I checked it's working perfectly. Because when floor = 0, then the code goes to line 66, and renders the down-arrow to go to only next page, we don't need the up-arrow there to go to the prev one. Right? Similarly when floor = 5, only we see the up-arrow not the down. What do you say?

Reply:
That means you are using down arrow to go to next page, i checked in figma tithi, i can see up arrow in when floor 0 and and in the last page only down arrow...Did you noticed that?

Reply:
You are so right @rajanibusani. Didn't notice well and also didn't get the concept before of going to previous floor with down arrow. Thank you very much. I have changed that now :)